packages/core/src/Stores/client-store.js

```diff
@@ -1639,8 +1639,9 @@ export default class ClientStore extends BaseStore {
             if (this.account_settings && !this.account_settings.residence) {
                 this.root_store.ui.toggleSetResidenceModal(true);
             }
-
-            await WS.authorized.cache.landingCompany(this.residence).then(this.responseLandingCompany);
+            if (this.residence) {
```

Comment:
this is to avoid input validation error for landing companies api 
<img width="505" alt="Screenshot 2023-04-12 at 11 30 17 AM" src="https://user-images.githubusercontent.com/84661147/231385182-807e8674-4ac1-45f4-a8ed-beeef7217226.png">